src/tc/tests/action_nat.rs

```diff
@@ -1,164 +0,0 @@
-// SPDX-License-Identifier: MIT
```

Comment:
Why this file is removed instead of amended?

Reply:
Hmm, I thought I moved the contents of this file into the same file as the other nat tests.

Likely that change was messed up in a rebase.  Will fix.

Reply:
The test in this file has been moved now.  It lives with the rest of the NAT tests